src/tiled/tileanimationeditor.cpp

```diff
@@ -257,6 +259,13 @@ const QVector<Frame> &FrameListModel::frames() const
 {
     return mFrames;
 }
+void FrameListModel::setDefaultFrameTime(int duration)
```

Comment:
There should be a gap between the above two lines.

Reply:
Thanks, Change has been made.